ui/packages/catalog-ui-search/src/main/webapp/component/search-form-interactions/search-form-interactions.view.js

```diff
@@ -161,8 +161,17 @@ module.exports =  Marionette.ItemView.extend({
             }));
             this.handleClick();
         },
-        isSystemTemplate: function() {
-            this.$el.toggleClass('is-system-template', this.model.get('createdBy') === 'system');
+    isProtectedTemplate: function () {
+        function isProtected(that) {
```

Comment:
I suspect people may want this function to be declared at the top- let me know if that is the case